### PR TITLE
add survos/crawler-bundle

### PR DESCRIPTION
**Pull request by tacman**
| Q             | A
| ------------- | ---
| License       | MIT
| Packagist     | https://packagist.org/packages/survos/crawler-bundle

### CODE REVIEW

File: survos/crawler-bundle/1.5/config/packages/survos_crawler.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+survos_crawler:
+# if you've run symfony proxy:domain:attach my-project, then the error messages will be displayed with this prefix.  That way, you can simply click on a link to open the page.
+#  base_url: 'https://my-project.wip/'
+    user_class: App\Entity\User
+    users:
+       - user@user.com
```
Comment (weaverryan) [inline survos/crawler-bundle/1.5/config/packages/survos_crawler.yaml:6]: Indentation is not quite right here

Comment (tacman) [reply survos/crawler-bundle/1.5/config/packages/survos_crawler.yaml:6]: OK, I tried it again, so that if you remove the # the spacing is correct.

Comment (weaverryan) [reply survos/crawler-bundle/1.5/config/packages/survos_crawler.yaml:6]: No - the problem is with this line precisely:

```suggestion
        - user@user.com
```

Comment (tacman) [reply survos/crawler-bundle/1.5/config/packages/survos_crawler.yaml]: Thanks!

**Review by weaverryan — APPROVED** [review]

**Pull request merged**